roles/benchmarking_deploy_coco_dataset/templates/pod.yml.j2

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: Pod
+metadata:
+ name: coco-dataset-downloader
+ namespace: {{ benchmarking_namespace }}
+spec:
+  restartPolicy: Never
+  nodeSelector:
+    kubernetes.io/hostname: {{ benchmarking_node_hostname}}
```

Comment:
`s/e}/e }/`

Reply:
+1
this one should have been caught by `ansible-lint` .... :/